make/go.mk

```diff
@@ -19,4 +19,12 @@ $(OUT_DIR)/operator:
 
 .PHONY: vendor
 vendor:
-	$(Q)go mod vendor
\ No newline at end of file
+	$(Q)go mod vendor
+
+TEKTON_INSTALLATION_CR_DIR=deploy/tekton
+
+.PHONY: generate-assets
+generate-assets:
+	@echo "generating assets bindata..."
+	@go install github.com/go-bindata/go-bindata/
+	@$(GOPATH)/bin/go-bindata -pkg tekton -o pkg/tekton/tekton_assets.go -nocompress -prefix $(TEKTON_INSTALLATION_CR_DIR) $(TEKTON_INSTALLATION_CR_DIR)
```

Comment:
Shall we enhance our `clean` target to clean this assets file right ??

Reply:
good point! Let me do that

Reply:
done in 45f0fdc